recipes/cusparselt/meta.yaml

```
@@ -0,0 +1,93 @@
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt\\types.h %LIBRARY_INC%\\cusparseLt              # [win64]
+    # - mkdir %LIBRARY_BIN%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.dll %LIBRARY_BIN%\\  # [win64]
+    # - mkdir %LIBRARY_LIB%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.lib %LIBRARY_LIB%\\  # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+  run_exports:
+    - {{ pin_subpackage('cusparseLt') }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+  run_constrained:
+    # Only GLIBC_2.17 or older symbols present
+    - __glibc >=2.17      # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/include/cusparseLt.h                    # [linux]
+    # - test -f $PREFIX/include/cusparseLt/types.h              # [linux]
+    - test -f $PREFIX/lib64/libcusparseLt.so                  # [linux]
+
+    # - if not exist %LIBRARY_INC%\\cusparseLt.h exit 1         # [win64]
+    # - if not exist %LIBRARY_INC%\\cusparseLt\\types.h exit 1  # [win64]
+    # - if not exist %LIBRARY_BIN%\\cusparseLt.dll exit 1       # [win64]
+    # - if not exist %LIBRARY_LIB%\\cusparseLt.lib exit 1       # [win64]
+
+about:
+  home: https://developer.nvidia.com/cusparselt
+  license: cuSPARSELt Software License Agreement
+  license_url: https://docs.nvidia.com/cuda/cusparselt/license.html
+  license_file: license.pdf
+  license_file: libcusparse_lt/libcusparse_lt/license.pdf
+  summary: "Tensor Linear Algebra on NVIDIA GPUs"
+  description: |
+    NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general
+    matrix-matrix operations in which at least one operand is a sparse matrix.
+    The cuSPARSELt APIs allow flexibility in the algorithm/operation selection,
+    epilogue, and matrix characteristics, including memory layout,
+    alignment, and data types.
+    License Agreements:- The packages are governed by the NVIDIA cuSPARSELt
+    Software License Agreement (EULA). By downloading and using the packages,
+    you accept the terms and conditions of the NVIDIA cuSPARSELt EULA -
+    https://docs.nvidia.com/cuda/cusparselt/license.html
+  doc_url: https://docs.nvidia.com/cuda/cusparselt/index.html
+  dev_url: https://developer.nvidia.com/cusparselt/downloads
+
+extra:
+  recipe-maintainers:
+    - mnicely
```

I have been helping maintain several NVIDIA packages, so I don't mind another 🙂
```suggestion
    - mnicely
    - leofang
```